.github/workflows/main.yml

```
@@ -0,0 +1,12 @@
+on: [push, pull_request]
+
+jobs:
+  stm32_ci_job:
+    runs-on: ubuntu-latest
+    name: CI job for stm32
+    steps:
+    - uses: actions/checkout@v2
```

This is a bit terse, but the YAML syntax (`-`) makes it clear I suppose. I had to do a double take not to confuse it with the step below.

Our other CI workflows are more explicit, like...
```suggestion
    - name: Checkout Code
      id: checkout
      uses: actions/checkout@v2
      
```

---

Right, I like the explicit naming better. I didn't quite grok it when I saw the yaml so thanks for highlighting it.